addons/website/views/snippets/s_adventure.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_adventure" name="Adventure">
+    <section class="s_adventure pt200 pb200 oe_img_bg" style="background-image: url('/web/image/website.s_adventure_default_image'); background-position: 50% 0%;">
```

Comment:
Why not fully centered? 🤔
```suggestion
    <section class="s_adventure pt200 pb200 oe_img_bg" style="background-image: url('/web/image/website.s_adventure_default_image'); background-position: 50% 50%;">
```

| very large screen |
|--------|
| ![image](https://github.com/user-attachments/assets/c84e0db5-54d4-402e-9fd5-22a6b85f3ac3) |
| ![image](https://github.com/user-attachments/assets/c5b4805d-419c-480d-94f6-d12fdd2fa234) |